docs/src/defaults.md

```diff
@@ -0,0 +1,85 @@
+## [Default Configurations](@id defaults)
+
+Sensible defaults have been chosen for the methodology to give broadly the best solution. The configurations can be revealed by adding the keyword
```

Comment:
Could be nice to add pointers to where each method is discussed. e.g., link the `TransformInversion` section to https://clima.github.io/EnsembleKalmanProcesses.jl/previews/PR392/ensemble_kalman_inversion/#etki. Not necessary though

Reply:
Done